appcell-resources/locales/en/translation.json

```diff
@@ -108,6 +111,7 @@
  ,"errorBoxStatusCommunication": "Communication failed. Specified Box does not exist or Please try again later."
  ,"errorBoxStatusNotFound": "Box does not exist."
  ,"errorFileFormat": "The file format is incorrect."
+ ,"errorGoogleLogin":  "I can not log in with this Google account. Please login with a different account and set up your Google Account."
```

Comment:
Please use the following message.  

Failed to log in using the provided Google account.  Please login with a proper account (Personium User Account) and register the Google account first.